ddtrace/ext/test.py

```diff
@@ -57,6 +60,12 @@
 # Test Codeowners
 CODEOWNERS = TEST_CODEOWNERS = "test.codeowners"
 
+# ITR Test skipping type
+ITR_TEST_SKIPPING_TYPE = "test.itr.tests_skipping.type"
+
+# ITR Test skipping type
```

Comment:
These comments don't add any information that isn't conveyed by the variable names. I'd suggest deleting them.